### PR TITLE
`ultralytics 8.3.197` New Construction-PPE dataset

### PR DESCRIPTION
**Pull request by RizwanMunawar**
@glenn-jocher Hi, this PR adds a new dataset, Construction-PPE in [Ultralytics docs](https://docs.ultralytics.com/). Thanks.

### Train

```python
from ultralytics import YOLO

# Load a model
model = YOLO("yolo11n.pt")  # load a pretrained model (recommended for training)

# Train the model
results = model.train(data="construction-ppe.yaml", epochs=100, imgsz=640)
```

```bash
yolo detect train data=construction-ppe.yaml model=yolo11n.pt epochs=100 imgsz=640
```

### Train batch Image

construction-ppe-dataset-sample

## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://github.com/ultralytics/actions)<sub>

### 🌟 Summary
Adds a new Construction-PPE dataset with docs, configs, and navigation, enabling easy training of YOLO11 models to detect worn and missing safety gear on construction sites. 🚧🦺

### 📊 Key Changes
- New dataset page: Construction-PPE documentation with structure, usage examples, sample image, FAQ, and citation.
- Dataset config added: `ultralytics/cfg/datasets/construction-ppe.yaml` with 11 classes (worn and missing PPE), train/val/test splits, and download URL.
- Docs navigation updates: Construction-PPE linked in datasets indexes and MkDocs nav.
- Minor cleanup: standardizes “New 🚀” label usage in dataset lists.
- Version bump: `__version__` updated from 8.3.196 to 8.3.197.

Helpful links:
- Construction-PPE docs: Construction-PPE Dataset
- YAML config: construction-ppe.yaml

### 🎯 Purpose & Impact
- Safety-focused AI made easier: Out-of-the-box dataset for detecting PPE presence and violations (e.g., no_helmet), ideal for compliance monitoring and alerts. ✅🚨
- Faster prototyping: Ready-to-train YAML and examples let users quickly train YOLO11 models for real-world construction scenarios.
- Broader applicability: Supports research, robotics, and workplace safety analytics with both compliant and non-compliant annotations.
- Improved discoverability: Clear docs and nav integration make the dataset easy to find and use.

### CODE REVIEW

**Comment by UltralyticsAssistant** [comment]
👋 Hello @RizwanMunawar, thank you for submitting an `ultralytics/ultralytics` 🚀 PR! This is an automated response to help speed things along — an Ultralytics engineer will review and assist you shortly. In the meantime, please review the checklist below to ensure a smooth merge:

- ✅ Define a Purpose: Clearly explain the purpose of your fix or feature in your PR description, and link to any relevant issues at the Ultralytics Issues page. Ensure your commit messages are clear, concise, and adhere to the project's conventions.
- ✅ Synchronize with Source: Confirm your PR is synchronized with the `ultralytics/ultralytics` `main` branch. If it's behind, update it by clicking the 'Update branch' button or by running `git pull` and `git merge main` locally.
- ✅ Ensure CI Checks Pass: Verify all Ultralytics Continuous Integration (CI) checks are passing. If any checks fail, please address the issues.
- ✅ Update Documentation: Update the relevant documentation for any new or modified features.
- ✅ Add Tests: If applicable, include or update tests to cover your changes, and confirm that all tests are passing.
- ✅ Sign the CLA: Please ensure you have signed our Contributor License Agreement if this is your first Ultralytics PR by writing "I have read the CLA Document and I sign the CLA" in a new message.
- ✅ Minimize Changes: Limit your changes to the minimum necessary for your bug fix or feature addition. "It is not daily increase but daily decrease, hack away the unessential. The closer to the source, the less wastage there is." — Bruce Lee

Additional quick notes for this dataset-focused PR:
- Please ensure the dataset YAML name used in docs and examples matches the file added in the repo.
- Confirm that the navigation entries and dataset index descriptions align with the new page title and content.
- Verify example commands use YOLO11 naming consistently in both Python and CLI snippets.
- Double-check license references and any download URLs match the intended release assets.

For more guidance, please refer to our Contributing Guide. Don’t hesitate to leave a comment if you have any questions. Thank you for contributing to Ultralytics! 🌟🛠️

**Review by glenn-jocher — APPROVED** [review]

**Comment by glenn-jocher** [comment]
@RizwanMunawar nice work!

**Comment by UltralyticsAssistant** [comment]
Huge congratulations on the merge, and heartfelt thanks to @RizwanMunawar for leading this and @glenn-jocher for the thoughtful contributions! “The future depends on what we do in the present.” — Mahatma Gandhi. This PR turns that spirit into impact by making safety-focused AI more accessible: clear docs, a ready-to-train dataset config, and seamless navigation to help the community quickly train YOLO11 models for PPE presence and violations on real construction sites.

Highlights we appreciate:
- New Construction-PPE docs and navigation for instant discoverability
- Ready-to-use config at `ultralytics/cfg/datasets/construction-ppe.yaml` with 11 classes
- Polished details and version bump to keep everything consistent

Quick start:
```python
from ultralytics import YOLO

model = YOLO('yolo11n.pt')
model.train(data='ultralytics/cfg/datasets/construction-ppe.yaml', epochs=100, imgsz=640)
```

Merged PR: see the details in the [Construction-PPE dataset pull request](https://api.github.com/repos/ultralytics/ultralytics/pulls/21976).

Thank you for advancing open, practical safety tooling—and for the craftsmanship that makes it effortless to use in Ultralytics HUB and beyond. 🚧🦺
